/**:
  ros__parameters:
    # clustering parameter
    angle_threshold: 0.174 # [rad] (10 deg)
    distance_threshold: 10.0 # [m]
    velocity_threshold: 4.0 # [m/s]

    # output object settings
    # set false if you want to use the object information from radar
    is_fixed_label: true
    fixed_label: "CAR"
    is_fixed_size: true
    size_x: 4.0 # [m]
    size_y: 1.5 # [m]
    size_z: 1.5 # [m]
